plan/stats.go

```diff
@@ -19,13 +19,15 @@ import (
 
 	"github.com/juju/errors"
 	"github.com/pingcap/tidb/expression"
+	"github.com/pingcap/tidb/statistics"
 	log "github.com/sirupsen/logrus"
 )
 
 // statsInfo stores the basic information of statistics for the plan's output. It is used for cost estimation.
 type statsInfo struct {
 	count       float64
 	cardinality []float64
+	colHist     []statistics.Column
```

Comment:
Seems that it is not used?

Reply:
Oh it's something should be done in next pr.